asado:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Velocity Controllers ---------------------------------------
  LF_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: LF_wheel_jnt
    pid: {p: 100.0, i: 0.01, d: 1.0}
  LB_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: LB_wheel_jnt
    pid: {p: 100.0, i: 0.01, d: 1.0}
  RF_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: RF_wheel_jnt
    pid: {p: 100.0, i: 0.01, d: 1.0}
  RB_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: RB_wheel_jnt
    pid: {p: 100.0, i: 0.01, d: 1.0}
  
  # PID gains
  gazebo_ros_control:
    pid_gains:
      LF_wheel_jnt: {p: 100.0, i: 0.01, d: 1.0}
      LB_wheel_jnt: {p: 100.0, i: 0.01, d: 1.0}
      RF_wheel_jnt: {p: 100.0, i: 0.01, d: 1.0}
      RB_wheel_jnt: {p: 100.0, i: 0.01, d: 1.0}